index.html

```diff
@@ -0,0 +1,889 @@
+        </nav>
+
+        <nav class="medium-screen">
+            <div class="flexbox">
+                <div class="flexbox-1">
+                    <div class="flexbox-1__box">
+                        <div class="box-1">
+                            <div></div>
+                        </div>
+
+                        <div class="box-2">
+                            <div class="section-wrapper-1">
+                                <div class="header-wrapper-1">
+                                    <ul>
+                                        <li><a href="#" class="active">News</a></li>
+                                        <li><a href="#">TNW2020</a></li>
+                                        <li><a href="#">Business</a></li>
+                                        <li><a href="">AMAs</a></li>
+                                        <li><a href="">Spaces</a></li>
+                                        <li><a href="">Terms &amp; Conditions</a></li>
+                                    </ul>
+                                    
+                                    <ul>
+                                        <li><a href="">About</a></li>
+                                        <li><a href="">Advetise</a></li>
+                                        <li><a href="">Jobs</a></li>
+                                        <li><a href="">Contact</a></li>
+                                    </ul>
+                                </div>  
+                            </div>
+
+                            <div class="section-wrapper-2">
+                                <div class="header-wrapper-2">
+                                    <div>
+                                        <ul>
+                                            <li><a href="#">LATEST</a> </li>
+                                            <li> <a href="">hard work</a></li>
+                                            <li> <a href="">plugged</a> </li>
+                                            <li> <a href="">Fundamentals</a> </li>
+                                            <li><a href="#">Readme</a> </li>
+                                        </ul>
+                                    </div>
+                                    
+                                    <div class="header-icon__img">
+                                        <img src="asset/images/search-icon.PNG" alt="Search Icon">
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                
+                <div class="flexbox-2">
+                    <a href="#">Did you know we have a TECH FESTIVAL? Check it out</a>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <main>
+        <section class="highlights">
+            <h6 class="hdr-heading"> Heading App</h6>
+            <div class="high-left high-1">
```

Comment:
- [ ] The image should be in the article tag itself. The article takes the whole height and width of the its background image.
The article is not composed only of its text. It's all that composes it in the context, as a self-containing unit. Read more about [article tags here. ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article)

![Screenshot_2020-03-25 dist html_css responsive_design md · master · Microverse guides tse code_review code_review_guidelines](https://user-images.githubusercontent.com/16193212/77533135-19205600-6e75-11ea-971c-f0af1bfd7db6.png)

----